.eslintrc.json

```diff
@@ -67,7 +67,19 @@
           },
           {
             "selector": "variable",
-            "format": ["camelCase", "PascalCase", "UPPER_CASE"]
+            "format": [],
+            "custom": {
+              "regex": "^_[a-zA-Z0-9]*$",
+              "match": true
+            }
+          },
+          {
+            "selector": "variable",
+            "format": ["camelCase", "PascalCase", "UPPER_CASE"],
+            "filter": {
+              "regex": "^_[a-zA-Z0-9]*$",
```

Comment:
```suggestion
              "regex": "^\w+$",
```

Reply:
```suggestion
              "regex": "^_[a-zA-Z]\w*$",
```

Reply:
No, we don't want to prohibit it. The dummy variable isn't always descriptive, and can improve code readability by "ignoring" its unnecessary meaning behind `_`